src/core.rs

```diff
@@ -0,0 +1,24 @@
+//! Hand written bindings for ostree-core.h
+
+use glib::VariantDict;
+
+/// The type of a commit object.
```

Comment:
For rustdoc rendering I think it would read better as a one-liner: `The type of $FOO: ($SIGNATURE)`